src/api/bot.ts

```diff
@@ -110,9 +112,9 @@ export default class Bot extends Client {
   async start(): Promise<void> {
     await this.login(this.config.token);
     this.logger.info("Initializing managers...");
-    Object.entries(this.managers).forEach(([k, v]) => {
-      v.init();
-    });
+    for (const manager of Object.values(this.managers)) {
+      await manager.init();
+    }
```

Comment:
Some managers depend on others to start up first (error handling, databases, etc. should be first)

Reply:
should probably leave a comment here